### DISCUSSION TITLE
how to create an account containing mnemonics

### DISCUSSION THREAD

**Discussion by leadsir** [seanjameshan/starknet.js#703] [Q&A] [answered]
I try the example that show on the official web,and it work to create an account and deploy it.But there is no mnemonics,as we konw, it is required to recovery an account with mnemonices in Argent x wallet.So,I want to know how to create mnemonics.Thanks.

**Comment by Svitln** [comment]
I want to dcide this too

**Comment by ivpavici** [comment]
hello! Starknet.js doesn't have any functionality regarding mnemonics, Argent-X uses ethers.js for this
https://github.com/argentlabs/argent-x/blob/099b22927759b8fdf3b52c67a72e9072ac1289a0/packages/extension/src/background/wallet.ts#L167

**Comment by tabaktoni** [comment, marked as answer]
https://github.com/PhilippeR26/starknet.js-workshop-typescript/blob/main/src/scripts/argentX/4.getPrivFromArgentXSeed.ts
